lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -120,24 +120,27 @@ def instructor_dashboard_2(request, course_id):
         'sales_admin': CourseSalesAdminRole(course_key).has_user(request.user),
         'staff': bool(has_access(request.user, 'staff', course)),
         'forum_admin': has_forum_access(request.user, course_key, FORUM_ROLE_ADMINISTRATOR),
-        'data_researcher': request.user.has_perm('student.can_research', course_key),
+        'data_researcher': request.user.has_perm('instructor.research', course_key),
```

Comment:
Is there a reason this shouldn't/can't be this?

`request.user.has_perm(permissions. CAN_RESEARCH, course_key)`